docker-compose.yml

```diff
@@ -18,6 +18,8 @@ services:
      - "8181:80"
     volumes:
       - mediawiki-images-data:/var/www/html/images
+    environment:
+     - MW_ELASTIC_HOST: 'elasticsearch.svc'
```

Comment:
This service is not defined in the docker-compose file.

Reply:
No, I think it is on line line 90.

There is a problem here though. It shouldn't have the - and that it making it fall over.